include/wx/msw/joystick.h

```diff
@@ -13,6 +13,8 @@
 
 #include "wx/event.h"
 
+class WXDLLIMPEXP_FWD_CORE wxJoystickThread;
```

Comment:
Is it necessary to export this? `wxJoystickThread` is only used internally in the `core` library.

Reply:
Maybe not? I added the export because it's there in the Unix implementation. I can test with that export removed on Monday.